man

```diff
@@ -0,0 +1,7 @@
+apiVersion: rabbitmq.com/v1beta1
+kind: RabbitmqCluster
+metadata:
+  name: sample0
+  namespace: rabbitmq-system
+spec:
+  replicas: 5
```

Comment:
Is that file accidentally committed?

Reply:
Yes, I will remove it. Thanks for catching it.